src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/FeatureOverviewEnvironment.tsx

```diff
@@ -39,37 +46,52 @@ const FeatureOverviewEnvironment = ({
         featureEnvironment => featureEnvironment.name === env.name
     );
 
-    const getOverviewText = () => {
-        if (env.enabled) {
-            return `${environmentMetric?.yes} received this feature because the following strategies are executing`;
-        }
-        return `This environment is disabled, which means that none of your strategies are executing`;
-    };
-
     return (
-        <div className={styles.featureOverviewEnvironment}>
+        <div
+            className={styles.featureOverviewEnvironment}
+            style={{
+                background: !env.enabled
+                    ? theme.palette.grey[100]
```

Comment:
We should try to avoid using specific shades of colors from the palette, it makes theming harder because when we switch theme we then have to swap out the grey color palette also. Is there a property in the theme that we could use that uses this shade of grey but abstracts better for theming?